tools/post_proofers/ppv_report.php

```diff
@@ -85,12 +85,13 @@
 
 // number of books post-processed by this PPer (including this one).
 $psd = get_project_status_descriptor('PPd');
-$result = mysqli_query(DPDatabase::get_connection(), "
+$sql = sprintf("
     SELECT COUNT(*) AS num_post_processed
     FROM projects
     WHERE $psd->state_selector
-      AND postproofer = '$project->postproofer'
-");
+    AND postproofer = LEFT('%s', 25)
```

Comment:
Please keep this indented from the `WHERE`, ideally by 4 spaces.

Reply:
And nice use of the `LEFT()`. 👍

Reply:
I see, I copy :)